src/main/kotlin/org/elm/ide/intentions/InlineDebugIntention.kt

```diff
@@ -24,45 +24,91 @@ class InlineDebugIntention : ElmAtCaretIntentionActionBase<InlineDebugIntention.
     override fun getFamilyName() = text
 
     override fun findApplicableContext(project: Project, editor: Editor, element: PsiElement): Context? {
-        element.ancestors.forEach {
-            when (it) {
+        element.ancestors.forEach { currentExpr ->
+            when (currentExpr) {
+                is ElmConstantTag -> {
+                    // A constant is unlikely to be interesting to debug.
+                    // It is more likely the context the constant is used in, that is interesting.
+                    currentExpr.parentOfType<ElmBinOpExpr>()
```

Comment:
I would like to establish some predictable rules for when the intention does what. I'm not sure the user finds it intuitive that using the intention on a literal will include a binary operation if one is present instead. Should this "promotion" also happen if the literal is used in a function call for example? Or should it just not happen?